apikey.go

```diff
@@ -76,7 +76,8 @@ func (k *ApiKey) IsCorrect(given string) error {
 
 	err := bcrypt.CompareHashAndPassword([]byte(k.HashedSecret), []byte(given))
 	if err != nil {
-		return err
+		return fmt.Errorf("hash does not match plaintext (hash: %s) (plaintext: %v...): %w",
```

Comment:
Alternatively (I'm fine with current):

```suggestion
		return fmt.Errorf("hash `%s` does not match plaintext `%v...`: %w",
```